Amplify/DefaultPlugins/AWSUnifiedLoggingPlugin/AWSUnifiedLoggingPlugin.swift

```diff
@@ -101,14 +103,16 @@ extension AWSUnifiedLoggingPlugin {
     }
     
     public func enable() {
+        enabled = true
         for (_, logger) in registeredLogs {
```

Comment:
do we need a thread safety mechanism here?  
i am not well versed with the entire feature but the setting of logger enabled values in a loop maybe requires some evaluation. in case this value is used in a concurrent manner.
similarly in `disable()`.